src/test/data/JsonSerializableDengueHotspotTrackerTest/invalidPersonDengueHotspotTracker.json

```
@@ -3,6 +3,7 @@
     "name": "Hans Muster",
     "postal": "948424a",
     "date": "2022-11-02",
-    "age": "21"
+    "age": "21",
+    "tag": "DENV1"
```

Is the format invalid on purpose?

```suggestion
    "tagged": [ "DENV1" ]
```